applications/luci-app-upnp/htdocs/luci-static/resources/view/status/include/80_upnp.js

```diff
@@ -67,7 +67,7 @@ return baseclass.extend({
 			];
 		});
 
-		cbi_update_table(table, rows, E('em', _('There are no active redirects.')));
+		cbi_update_table(table, rows, E('em', _('There are no active port mappings.')));
```

Comment:
For example which languages?

Reply:
In any case, what language does miniupnpd itself use for opened ports?

Reply:
Ok, what about "forwarding"? It's still may be difficult to translate but this is more precise term as for me.

> For example which languages?

Probably all. The mapping translates to "cartography". The most similar word is "accordance" but it's more abstract.

Reply:
I prefer mapping especially if the original specs use this terminology. Pinhole makes sense for ipv6. Port forwarding is acceptable since that's what firewall uses, I think.

Reply:
Port mapping sounds strange to me. Forwarding is more explanatory from end-user perspective

Reply:
The English language has its idiosyncracies. But mapping sounds great to my native ears.  Forward seems to be preferred then. Can we do that @Self-Hosting-Group ?